src/customize-pop/accessibility-settings.md

```diff
@@ -0,0 +1,125 @@
+# Accessibility Settings
+
+Configure accessibility settings in `Settings` ➞ `Accessibility`. Accessibility features provide sensory enhancement to users performing input or observing visual and audible output. This section briefly describes each setting, but more information can be found on [GNOME's help page](https://help.gnome.org/users/gnome-help/stable/a11y.html).
+
+![Accessibility Settings](/images/accessibility-settings/accessibility-settings.png)
+
+## Always Show Accessibility Menu
+
+When `Always Show Accessibility Menu` is enabled, accessibility options can be toggled using an icon in the upper right corner.
+
+![Always Show Menu](/images/accessibility-settings/always-show-menu.png)
+
+## Seeing
+
+`Seeing` settings provide visual enhancements to help visually impaired users navigate the operating system's UI.
+
+### High Contrast
+
+Enabling `High Contrast` applies a high contrast theme to system windows, application windows, and their icons.
+
+![High Contrast](/images/accessibility-settings/high-contrast.png)
+
+### Large Text
+
+Enabling `Large Text` increases the text size of system windows and applications.
```

Comment:
I might tweak 
`increases the text size of system windows and applications`
to
`increases the size of text across system windows and supported applications`
Some applications will likely not play nice with this feature, so adding something to imply that should prime the user to consider the possibility if an expected response isn't seen.